### ISSUE TITLE
Demo crash when click ArekMicrophone

### ISSUE THREAD

**Issue by Huang-Libo** [ennioma/arek#47]
```
fatal error: unexpectedly found nil while unwrapping an Optional value
```
![2016-12-18 14 45 31](https://cloud.githubusercontent.com/assets/12227130/21292019/c374fd5a-c530-11e6-81be-a97c75d26235.png)

**Comment by ennioma** [comment]
Hi @Huang-Libo,
I'm not able to reproduce this.

Could you please provice me more details?

**Comment by Huang-Libo** [comment]
I run the demo on iPhone7 Plus(not simulator), iOS 10.2

**Comment by ennioma** [comment]
Ok thanks,
I'll take a look as soon as possible!

**Comment by evermeer** [comment]
Isn't this fixed with https://github.com/ennioma/arek/commit/a5b073d74c3486a13918f7101fba0b54280942dd

**Comment by ennioma** [comment]
Yes @evermeer, it should but In the meanwhile I'd test it on a device.

**Comment by ennioma** [comment]
This should be fixed @Huang-Libo

**Comment by gerchicov-bp** [comment]
still appears

**Comment by evermeer** [comment]
NSMicrophoneUsageDescription is missing again from the demo

**Comment by ennioma** [comment]
Should be definitely closed with #71
